aiinfra-cluster/installation_scripts/install_cloud_ops_agent.sh

```diff
@@ -236,6 +251,7 @@ main() {
         fail "Legacy or Ops Agent is already installed."
     fi
 
+    install_wget_with_retry
     echo >&2 "Install Ops Agent"
     retry_with_backoff 10 32 install_opsagent \
         || { echo >&2 'Failed to install Ops Agent'; return 1; }
```

Comment:
the function call in the first line here should probably match the way the next three lines call install_opsagent since the way this is currently written (since the function ends with has `return 1` and not `exit 1`), even if it fails to install wget, it will move onto installing ops agent

Reply:
addressed in new iteration.